Sources/MapboxMaps/Foundation/MapView.swift

```diff
@@ -34,6 +35,13 @@ open class MapView: UIView {
     /// Controls the addition/removal of annotations to the map.
     public internal(set) var annotations: AnnotationOrchestrator!
 
+    /// Property that describes the current language for `SymbolLayer.textField`
+    public var locale: Locale = Locale.current {
+        didSet {
+            mapboxMap.style.localizeLabels(into: locale)
+        }
+    }
```

Comment:
Could this be an unsupported Locale?

Reply:
It can, but in that case, we will default to the local case if there is no data that exists

Reply:
We may want to make that behavior more explicit.

Reply:
I could add a check when passed in to see if it is under one of our supported languages. If so, then continue, otherwise don't do anything